nixos/modules/services/x11/display-managers/slim.nix

```diff
@@ -128,7 +128,7 @@ in
 
     # Allow null passwords so that the user can login as root on the
     # installation CD.
-    security.pam.services.slim = { allowNullPassword = true; startSession = true; };
+    security.pam.services.slim = { allowNullPassword = true; startSession = true; pamMount = true; };
```

Comment:
Isn't pamMount enabled when security.pam.mount.enable is enabled? That true is forcing it, isn' it?